### PR TITLE
Add mapping for Xamarin.CommunityToolkit

### PR DESCRIPTION
**Pull request by etvorun**
Add mapping for Xamarin.CommunityToolkit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -155,6 +155,25 @@ The above example would transform all occurrences of `Windows.UI.WindowManagemen
...
+      "Xamarin.CommunityToolkit.UI.Views"
+    ]
+  }
+}
+```
+
+Upgrade Assistant will update XML namespaces like `xmlns:toolkit="http://xamarin.com/schemas/2020/toolkit"` to `xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"`.
+It will also update XML namespaces like `xmlns:tk='clr-namespace:Xamarin.CommunityToolkit.Behaviors; assembly=...'` to `xmlns:tk='clr-namespace:Microsoft.Maui.Behaviors; assembly=...'`.
+
 ## Engage, Contribute and Give Feedback
```
Comment (jstedfast) [inline README.md:177]: Would it make sense to modify the UA logic to use the existing "namespace" mappings to remap these? Or does this provide additional info that is needed?

Comment (jstedfast) [reply README.md:177]: Oh, I guess it adds "properties"

**Review by jstedfast — APPROVED** [review]

**Comment by etvorun** [comment]
NOT READY TO MERGE

File: README.md [hunk 2]
```diff
@@ -155,6 +155,25 @@ The above example would transform all occurrences of `Windows.UI.WindowManagemen
...
+      "Xamarin.CommunityToolkit.Behaviors",
+      "Xamarin.CommunityToolkit.Converters",
+      "Xamarin.CommunityToolkit.UI.Views"
+    ]
+  }
+}
+```
+
+Upgrade Assistant will update XML namespaces like `xmlns:toolkit="http://xamarin.com/schemas/2020/toolkit"` to `xmlns:toolkit="http://schemas.microsoft.com/dotnet/2022/maui/toolkit"`.
+It will also update XML namespaces like `xmlns:tk='clr-namespace:Xamarin.CommunityToolkit.Behaviors; assembly=...'` to `xmlns:tk='clr-namespace:Microsoft.Maui.Behaviors; assembly=...'`.
```
Comment (mgoertz-msft) [inline README.md:175]: Should we explain that `properties` contains the CLR namespaces that were part of the old xmlns?

**Review by mgoertz-msft — APPROVED** [review]

**Review by mgoertz-msft — APPROVED** [review]

**Review by jstedfast — APPROVED** [review]
Might be worth including the apimap/packagemap in UA itself for CommunityToolkit, no?

**Pull request merged**